.github/workflows/publish-swaggerhub.yaml

```diff
@@ -0,0 +1,83 @@
+#
+#  Copyright (c) 2023 Mercedes-Benz Tech Innovation GmbH
+#  Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+
+---
+name: "Publish OpenAPI to Swaggerhub"
+
+on:
+  workflow_call:
+  push:
+    branches:
+      - feature/swagger-api-2
+
+jobs:
+  swagger-api:
+    runs-on: ubuntu-latest
+    env:
+      SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
+      SWAGGERHUB_USER: ${{ secrets.SWAGGERHUB_USER }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup JDK 17
+        uses: actions/setup-java@v3.11.0
+        with:
+          java-version: '17'
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+
+      - name: Install tools
+        run: |
+          npm i -g swaggerhub-cli
+          sudo apt install -y crudini
+
+      - name: Extract versions
+        run: |
+          export DOWNSTREAM_VERSION=$(crudini --get ./gradle.properties '' version)
```

Comment:
no need for `crudini`, just use 
```bash 
grep "version" gradle.properties  | awk -F= '{print $2}')
```

Reply:
I considered that. But that approach is not aware of sections (and quotes, but that's less important), which our .ini files use. I know it's one more apt call, but it's worth it.

Reply:
Sections are not relevant for the task at hand, neither is writing, updating or deleting lines from the file.
We are not using 99% of what that tool offers, and installing random tools for a task that can be solved with standard tools is *not* the way to go.
Thats not even mentioning all the issues third-party tools have in general (api changes, lifecycle, potential security holes,...).

Please get rid of `crudini` and use standard CLI tools.

Reply:
True, that is EDC policy. I'll do it with preinstalled tools instead.